From fa181f8768c9471e12459a4f9d074f37f10bb505 Mon Sep 17 00:00:00 2001
From: chromium-autoroll <chromium-autoroll@skia-public.iam.gserviceaccount.com>
Date: Thu, 20 Jul 2023 00:29:47 +0000
Subject: [PATCH] Roll WebRTC from dbb89430ef77 to e9e03a916050 (3 revisions)

https://webrtc.googlesource.com/src.git/+log/dbb89430ef77..e9e03a916050

2023-07-19 joachimr@meta.com Fix inaccurate contentType in RTCInbound/OutboundRtpStreamStats
2023-07-19 phancke@microsoft.com Prevent SDP munging of duplicate SSRCs
2023-07-19 chromium-webrtc-autoroll@webrtc-ci.iam.gserviceaccount.com Roll chromium_revision 21b76e39ae..58a3c40eba (1172261:1172400)

If this roll has caused a breakage, revert this CL and stop the roller
using the controls here:
https://autoroll.skia.org/r/webrtc-chromium-autoroll
Please CC webrtc-chromium-sheriffs-robots@google.com,webrtc-infra@google.com on the revert to ensure that a human
is aware of the problem.

To file a bug in WebRTC: https://bugs.chromium.org/p/webrtc/issues/entry
To file a bug in Chromium: https://bugs.chromium.org/p/chromium/issues/entry

To report a problem with the AutoRoller itself, please file a bug:
https://bugs.chromium.org/p/skia/issues/entry?template=Autoroller+Bug

Documentation for the AutoRoller is here:
https://skia.googlesource.com/buildbot/+doc/main/autoroll/README.md

Bug: chromium:1459124
Tbr: webrtc-chromium-sheriffs-robots@google.com
Change-Id: I2340d48bb0484a1dd608eb61bb97de4c3313307b
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4703308
Bot-Commit: chromium-autoroll <chromium-autoroll@skia-public.iam.gserviceaccount.com>
Commit-Queue: chromium-autoroll <chromium-autoroll@skia-public.iam.gserviceaccount.com>
Cr-Commit-Position: refs/heads/main@{#1172688}
---

diff --git a/DEPS b/DEPS
index 0667eab6..f115f2e 100644
--- a/DEPS
+++ b/DEPS
@@ -1923,7 +1923,7 @@
     Var('chromium_git') + '/external/github.com/gpuweb/cts.git' + '@' + '954ac6b88b2b197226503b74a32fc29884ebd856',
 
   'src/third_party/webrtc':
-    Var('webrtc_git') + '/src.git' + '@' + 'dbb89430ef772ebaf8a0170b0be150129eb9fd63',
+    Var('webrtc_git') + '/src.git' + '@' + 'e9e03a9160502b23749e8f64da517f83457e2ad3',
 
   # Wuffs' canonical repository is at github.com/google/wuffs, but we use
   # Skia's mirror of Wuffs, the same as in upstream Skia's DEPS file.
